packages/integrations/tailwind/src/index.ts

```diff
@@ -23,15 +24,22 @@ async function getPostCssConfig(
 
 async function getViteConfiguration(
 	tailwindConfigPath: string | undefined,
-	viteConfig: UserConfig
+	nesting: boolean,
```

Comment:
We're starting to have lots of options here, could it be better to have a single object instead?

Reply:
Just an internal function so not a big deal. Going to merge this in but feel free to refactor if you'd like!